File: reference/CodeExamples/java/src/com/db4odoc/identiy/Pilot.java
package com.db4odoc.identiy;


class Pilot {
    private String name;

    public Pilot(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Pilot{" +
                "name='" + name + '\''+
                '}';
    }
}

File: db4o.net/Db4oTutorial/Db4odoc.Tutorial.Chapters/F1/Chapter1/Pilot.cs
namespace Db4odoc.Tutorial.F1.Chapter1
{
    public class Pilot
    {
        string _name;
        int _points;
        
        public Pilot(string name, int points)
        {
            _name = name;
            _points = points;
        }
        
        public string Name
        {
            get
            {
                return _name;
            }
        }
        
        public int Points
        {
            get
            {
                return _points;
            }
        }   
        
        public void AddPoints(int points)
        {
            _points += points;
        }    
        
        override public string ToString()
        {
            return string.Format("{0}/{1}", _name, _points);
        }
    }
}